content/docs/latest/spire-helm-charts-hardened-about/identifiers.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Identifiers
+short: Identifiers
+description: How to configure SPIFFE IDs
+kind: spire-helm-charts-hardened-about
+weight: 200
+aliases:
+    - /docs/latest/helm-charts-hardened/identifiers
+---
+
+## Defaults
+
+By default the chart deploys the SPIRE Controller Manager that enables management of SPIFFE Identifiers by Kubernetes Custom Resources.
+
+The chart itself by default deploys a ClusterSPIFFEID Custom Resource that gives an identifier to all pods.
+
+Out of the box, in a lot of use cases you do not need to add additional identifiers.
+
+## Custom / Additional Identifiers
+
+But, in some cases you may want to customize your identifiers or add additional ones.
+
+While you can manage identities using the Kubernetes Custom Resources via https://github.com/spiffe/spire-controller-manager?tab=readme-ov-file#custom-resources directly
+we do not recommend doing that as it takes care to not misconfigure.
+
+The chart supports managing the Custom Resources from the charts values file for even easier management. We recommend using that mechanism.
```

Comment:
Suggest to lead with what is supported. Maybe not even mentioned the unsupported. manual path.

Reply:
Was trying to avoid the statement "you never told me that". But can remove it if you feel its better not to even bring it up.

Reply:
You can keep it in. More important is to lead with what is supported.

Reply:
Ok. I reordered things.